packages/widget-notes/src/Widget.tsx

```diff
@@ -17,7 +24,10 @@ import '../css/react-splitter-layout.css'
 import NoteContext, { NoteProvider } from './Context'
 import NoteEditor from './components/Editor'
 import NoteListItem from './components/ListItem'
-import { Note } from './types'
+import { Events, Note } from './types'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCloud } from '@fortawesome/free-solid-svg-icons'
```

Comment:
While not explicitly documented or defined as EsLint rule, let's move imports of external packages at the top and then imports of internal modules